pkgs/applications/networking/browsers/chromium/update.nix

```diff
@@ -64,7 +64,7 @@ in rec {
     csv2nix = name: src: import (runCommand "${name}.nix" {
       src = builtins.fetchurl src;
     } ''
-      esc() { echo "\"$(echo "$1" | sed -e 's/"\\$/\\&/')\""; }
+      esc() { echo "\"$(echo "$1" | sed -e 's/"\\$/\\&/')\""; } # ohai emacs "
```

Comment:
Could the Nix syntax be fixed upstream? Just wondering because I have seen these comments in various places now...